test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +11,17 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // Null objects
+        assertAnyNull(null, null, null);
+        assertAnyNull(42, null, "red bull");
+
+        // No null
+        assertAllNotNull(42, 43, 44, 45);
+        assertAllNotNull(new int[] {1,2,3,4}, "NOT NULL");
```

Comment:
Generally ok, but perhaps you need to cover more of the scenarios. (e.g. empty list/no arguments)